difflint/lib.py

```diff
@@ -2,24 +2,44 @@
 from datetime import datetime
 from difflib import unified_diff
 from io import StringIO
-from os import unlink
+from os import getcwd, unlink
+import os.path
```

Comment:
nit: Is there a reason you chose to import path this way instead of doing `from os import getcwd, path, unlink` on the line above?